app/views/search/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="search-results container">
+
+  <div class="row justify-content-between">
+    <div class="col title-section">FILTER BY</div>
+    <div class="col body-toned-down text-right">Clear All</div>
+  </div>
+  <div class="row">
+    <div class="col mt-2">
+      <div class="btn btn__filter--white button-dark ml-0 mr-2">Badge</div>
+      <div class="btn btn__filter--white button-dark ml-2">Category</div>
+    </div>
+  </div>
+
+  <% if @users.present? %>
+    <div class="row justify-content-between mt-3">
+      <div class="col title-section">PEOPLE</div>
+      <div class="col body-toned-down text-right">See All</div>
+    </div>
+
+    <div class="row mt-2">
+      <% @users.each do |result| %>
+        <div class="row col-12 col-lg-3">
+          <div class="col-2 text-left">
+            <%= image_tag("home-profile_01.jpg", alt: "rss feed", class: "rounded-circle avatar") %>
```

Comment:
can we put either a TODO to change this or have duo provide a default avatar User & Organization

Reply:
gonna add faker profile images so they will all have one